### PR TITLE
Cleanup pom for release

### PR DESCRIPTION
**Pull request by jtnord**
Just some pom cleanup as the previous releases have not updated the POM after release :-(

@reviewbybees

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -1,41 +1,57 @@
...
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>1.609</version>
+    <version>2.24</version>
   </parent>
 
   <artifactId>distfork</artifactId>
-  <version>1.4-SNAPSHOT</version>
+  <version>1.4.2-SNAPSHOT</version>
```
Comment (oleg-nenashev) [inline pom.xml]: Maybe 1.5 taking the core dependency bump?

**Review by oleg-nenashev — APPROVED** [review]

**Comment by ghost** [comment]
This pull request originates from a [CloudBees](https://www.cloudbees.com/) employee. At CloudBees, we require that all pull requests be reviewed by other CloudBees employees before we seek to have the change accepted. If you want to learn more about our process please see [this explanation](https://github.com/reviewbybees/about#about-reviewbybees).

**Comment by oleg-nenashev** [comment]
:bee: excepting the version number. Core dependency is also bumped, but I suppose it is fine

File: pom.xml (hunk 1 above)
Comment (jtnord) [reply pom.xml]: done

File: pom.xml [hunk 2]
```diff
@@ -1,41 +1,57 @@
...
+  <description>Turns a Jenkins cluster into a general purpose batch job execution environment through an SSH-like CLI.
+               This plugin adds a new command 'distfork' to Jenkins CLI, which can be used to execute arbitrary command
+               on a agent of your choice. The distfork command is modeled after ssh, but it's Jenkins aware — for example,
+               instead of hardcoding a machine name, you can specify a label to let Jenkins choose a agent.
+               This opens up a Jenkins cluster for doing all kinds of things without requiring a job/build notion.
+  </description>
   <url>http://wiki.jenkins-ci.org/display/JENKINS/DistFork+Plugin</url>
 
+  <properties>
+    <findbugs.failOnError>false</findbugs.failOnError>
```
Comment (jtnord) [inline pom.xml:24]: currently a few pre-existing FindBugs issues.  Not planning on fixing them here.

**Comment by oleg-nenashev** [comment]
re-:bee:

**Review by batmat — APPROVED** [review]
LGTM :bee:

**Pull request merged**